card/lib/generators/card/templates/create_cards_tables.erb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
```

Comment:
Looks like this thought got cut off :)

> What if they already have a table named 'card', for example?

Feels like you're getting lost in bogus edge cases. I doubt this will come up, and when it does, the namespace issues are coing to be much bigger than table names.

We need to get an engine working today or tomorrow and not keep wandering in the wilderness. This prefix stuff is wilderness.

> Is there any reason for adding a MyDeck::Engine?

Yes.
1. If we keep the Engine configuration closely parallel to the Application configuration, we'll make it easy for someone to go from a rails deck to and app deck; this is a case we want to support.
2. The boot process is set up so that many things happen when the engine definition is complete.  Assigning configuration values afterwards means they will get ignored by anything that happens in the mean time.
3. I think there's actually a significant error in setting the engine root equal to Rails.root.  Typically, I think the engine root is elsewhere, and this is determined by where the Engine itself is defined.  If it's defined in the gem, the engine root will by default be in the gem; that's not what we want.
4. If you don't create a new engine class, you have to create a whole new pattern for how everything works.  Unlike with code inside our gems, in code in other people's rails apps we definitely want to follow railsy patterns as closely as makes sense for Wagn.